packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -330,8 +338,6 @@ class TechnologyController {
 			await trx.rollback();
 			throw error;
 		}
-		technology.likes = 0;
-		technology.status = technologyStatuses.DRAFT;
 		this.indexToAlgolia(technology);
```

Comment:
@alexandreadames isso deve mesmo ser chamado aqui? 👀
cc @lcnogueira

Reply:
@mateus4k o indexToAlgolia só deve ser chamdo na hora que a tecnologia for para published, no store ela nunca vai ser published, teria que ver o local onde o admin publica.

Reply:
O admin só altera o status usando a rota `technologies/:id/update-status`
os restante dos eventos são automáticos certo?

Reply:
Isso @NaylsonFerreira